broker/src/tunneldigger_broker/broker.py

```diff
@@ -37,7 +40,10 @@ def __init__(
         self.tunnel_ids = set(range(tunnel_id_base, tunnel_id_base + max_tunnels))
         self.tunnels = {}
         self.last_tunnel_created = None
+        self.last_tunnel_created_per_ip = {}
```

Comment:
Please describe somewhere what the data stored in this dict looks like. (Gosh I wish this was written in a typed language...)